src/webgpu/api/validation/resource_usages/textureUsageInRender.spec.ts

```diff
@@ -196,14 +296,29 @@ g.test('subresources_and_binding_types_combination_for_color')
           type1: 'render-target',
           _usageSuccess: false,
         },
+        {
+          type0: 'render-target',
+          type1: 'render-target',
+          _usageSuccess: false,
+        },
       ] as const)
+      // Every color attachment can use only one single subresource. In addition, all color
+      // attachments' size should be the same.
```

Comment:
uNit: Could be two separate `.unless()` calls with two separate comments, just to be a little clearer.

Reply:
Done